src/components/MultiHopTrade/components/TradeConfirm/TradeConfirm.tsx

```diff
@@ -157,21 +157,21 @@ export const TradeConfirm = () => {
   const getSellTxLink = useCallback(
     (sellTxHash: string) =>
       getTxLink({
-        name: tradeQuoteStep?.sources[0]?.name,
+        name: tradeQuoteStep?.source,
```

Comment:
tyvm 🧠  was irking me why we were using the ZRX response shape here, even though we never consumed `proportion` and it's only a ZRX thing